styles/main/_form.scss

```diff
@@ -0,0 +1,373 @@
+		border-bottom-color: $colorFormElementAccent;
+	}
+
+	&.error {
+		border-bottom-color: $colorFormElementError;
+	}
+}
+
+input[type="checkbox"] {
+	top: 2px;
+	height: 16px;
+	width: 16px;
+	appearance: none;
+	color: $colorFormElementAccent;
+	border: none;
+	border-radius: 3px;
+	box-shadow: 0 0 0 1px black(0.7);
+
+	&::before {
+		content: "✔";
+		position: absolute;
+		text-align: center;
+		font-size: 16px;
+		line-height: 16px;
+		top: 0;
+		bottom: 0;
+		left: 0;
+		right: 0;
+		width: auto;
+		height: auto;
+		// Use `visibility` instead of `display` to avoid layout recalculation
+		visibility: hidden;
+	}
+
+	&:checked::before {
+		// Use `visibility` instead of `display` to avoid layout recalculation
+		visibility: visible;
+	}
+}
+
+input[type="checkbox"].slider {
+	top: 5px;
+	height: 22px;
+	width: 42px;
+	border: 1px solid rgba(0, 0, 0, 0.2);
+	box-shadow: 0 1px 0 rgba(255, 255, 255, 0.02);
+	border-radius: 11px;
+	background: $colorFormElementBg;
+
+	&::before {
+		content: "";
+		background-color: $colorFormElementAccent;
+		height: 14px;
+		width: 14px;
+		left: 3px;
+		top: 3px;
+		border: none;
+		border-radius: 7px;
+		visibility: visible;
+	}
+
+	&:checked {
+		background-color: $colorFormElementAccent;
+	}
+
+	&:checked::before {
+		left: auto;
+		right: 3px;
+		background-color: $colorFormElementFg;
+	}
+}
+
+div.select {
+	font-size: 12px;
+	background: $colorFormElementBg;
+	border-radius: 3px;
+	border: 1px solid black(0.2);
+	box-shadow: 0 1px 0 white(0.02);
+
+	&::after {
+		position: absolute;
+		content: "≡";
+		right: 8px; // must match the right padding of the `<select>` element
+		top: 3px; // must match the top padding of the `<select>` element
+		color: $colorFormElementAccent;
+		font-size: 16px;
+		font-weight: bold;
+		pointer-events: none;
+	}
+}
+
+select {
+	padding-left: 8px;
+	padding-right: 8px; // must match the right position of the `div.select::after`
+	appearance: none;
+	background: none;
+}
+
+select option {
+	padding: 2px 0;
+	transition: none;
+}
+
+// 2. Input groups
+//
+// Only <div> are expected as direct children of a form.
+// These <div> constitute a so-called "input group".
+// An input group consists of the following elements in the prescribed
+// order:
+//
+//  - an optional `<label>`
+//  - exactly one mandatory HTML form element:
+//      -- `<input>`,
+//      -- `<div class='select`> nesting a `<select>`,
+//      -- `<textarea>`, or
+//      -- `<output>`
+//  - zero or more optional `<p>` elements with a descriptive text
+//  - zero or more optional `<div>` with nested input groups
+//
+// One note about `<select>`: This element is hard to style with CSS,
+// because it is treated as a replaced element. Hence, `<select>` needs to
+// be wrapped into an additional `<div>` and most styles are applied to
+// this `<div>`.
+// See: https://developer.mozilla.org/en-US/docs/Learn/Forms/Advanced_form_styling
+//
+// An input group can have one out of four layouts which determines
```

Comment:
```suggestion
// An input group can have one out of four layouts which deteriine
```

Reply:
Right, a typo in a suggestion :flushed:.

Reply:
Fixed in https://github.com/LycheeOrg/Lychee-front/pull/322/commits/9e941bbfbc26240a8b3e6bfa63e7546fe59123be.